registry/emailTemplates.xml

```diff
@@ -70,4 +69,7 @@
 	<email key="ANNOUNCEMENT" subject="emails.announcement.subject" body="emails.announcement.body"/>
 	<email key="EDITORIAL_REMINDER" can_disable="0" from_role_id="16" to_role_id="1048576" subject="emails.editorialReminder.subject" description="emails.announcement.description" body="emails.editorialReminder.body"/>
 	<email key="DISCUSSION_NOTIFICATION" subject="discussion.notification.subject" body="discussion.notification.body"/>
+	<email key="VERSION_CREATED" subject="emails.versionCreated.subject" body="emails.versionCreated.body"/>
+	<email key="REVIEW_COMPLETE" subject="emails.reviewComplete.subject" body="emails.reviewComplete.body"/>
+	<email key="REVIEW_EDITED" subject="emails.reviewEdited.subject" body="emails.reviewEdited.body"/>
```

Comment:
Call this `REVIEW_EDIT` so it has the same verb tense as the other review emails. You can move these two up with the other `REVIEW_*` emails too, just to keep them together.